src/test/java/service/impl/FruitTransactionParserImplTest.java

```diff
@@ -0,0 +1,56 @@
+package service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.FruitTransaction;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import service.FruitTransactionParserService;
+import service.ReaderDataService;
+
+public class FruitTransactionParserImplTest {
+    private static final String PATH_TO_INPUT_FILE =
+            "src/test/resources/InputDataForFruitTransactionParser.csv";
+    private static List<String> DATA_FROM_FILE;
+    private static List<FruitTransaction> EXPECTED_AFTER_PARSE;
+    private static ReaderDataService readerDataService;
```

Comment:
same - you shouldn't use your reader
please create yourself a List<String> that you will pass to your parser